Source/Core/Common/BitUtils.h

```diff
@@ -8,6 +8,7 @@
 #include <climits>
 #include <cstddef>
 #include <cstring>
+#include <cstdint> // include this header for uint64_t
```

Comment:
This should go above `<cstring>`, we also don't use comments to indicate what we use from the standard header

Reply:
@lioncash Thanks, it's already corrected.